README.md

```diff
@@ -1,55 +1,72 @@
+[comment]: # ( Copyright Contributors to the Open Cluster Management project )
+
 # Governance Policy Addon Controller
 
+Open Cluster Management - Governance Policy Addon Controller
+
+[![Build](https://img.shields.io/badge/build-Prow-informational)](https://prow.ci.openshift.org/?repo=stolostron%2Fgovernance-policy-addon-controller)
+[![KinD tests](https://github.com/stolostron/governance-policy-addon-controller/actions/workflows/kind.yml/badge.svg?branch=main&event=push)](https://github.com/stolostron/governance-policy-addon-controller/actions/workflows/kind.yml)
 [![License](https://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
 
-## From the kubebuilder quickstart:
+## Description
 
-Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
-```bash
-make run
-```
+The governance policy addon controller manages installations of policy addons on managed clusters by using [ManifestWorks](https://github.com/stolostron/api/blob/master/work/v1/types.go). The addons can be enabled, disabled, and configured via their `ManagedClusterAddon` resources. For more information on the addon framework, see the [addon-framework enhancement/design](https://github.com/open-cluster-management-io/enhancements/tree/main/enhancements/sig-architecture/8-addon-framework). The addons managed by this controller are:
```

Comment:
Would it make sense to link ManifestWork to the upstream instance? There's also a doc file, which may be more informative: https://github.com/open-cluster-management-io/api/blob/main/docs/manifestwork.md

Reply:
Great suggestions!